From b9e12b89dcac86ec025f43ade58cf4b33a4a58af Mon Sep 17 00:00:00 2001
From: kok-o3 <ko282ko82@gmail.com>
Date: Wed, 18 Jun 2025 14:37:56 +0530
Subject: [PATCH] include: fix time.h inclusion

usr/include/linux/videodev2.h:2458:20: error: field has incomplete type 'struct timespec'
 2458 |         struct timespec                 timestamp;
      |                                         ^
usr/include/linux/videodev2.h:2458:9: note: forward declaration of 'struct timespec'
 2458 |         struct timespec                 timestamp;
      |                ^
1 error generated.

Signed-off-by: kok-o3 <ko282ko82@gmail.com>
---
 include/uapi/linux/videodev2.h | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/include/uapi/linux/videodev2.h b/include/uapi/linux/videodev2.h
index 91ede37556f3e..8ab04d29655a9 100644
--- a/include/uapi/linux/videodev2.h
+++ b/include/uapi/linux/videodev2.h
@@ -57,9 +57,7 @@
 #ifndef _UAPI__LINUX_VIDEODEV2_H
 #define _UAPI__LINUX_VIDEODEV2_H
 
-#ifndef __KERNEL__
-#include <sys/time.h>
-#endif
+#include <linux/time.h>
 #include <linux/compiler.h>
 #include <linux/ioctl.h>
 #include <linux/types.h>
